client/my-sites/domains/domain-management/contacts-privacy/index.jsx

```diff
@@ -55,31 +58,40 @@ const ContactsPrivacy = React.createClass( {
 						hasPrivacyProtection={ hasPrivacyProtection }
 						privateDomain={ privateDomain }
 						privacyAvailable={ privacyAvailable }
-						currentUserCanManage={ currentUserCanManage } />
+						currentUserCanManage={ currentUserCanManage }
+					/>
 
 					<VerticalNavItem
-							path={ paths.domainManagementEditContactInfo( this.props.selectedSite.slug, this.props.selectedDomainName ) }>
-						{ this.translate( 'Edit Contact Info' ) }
+						path={ paths.domainManagementEditContactInfo(
+							this.props.selectedSite.slug,
+							this.props.selectedDomainName
+						) }
+					>
+						{ translate( 'Edit Contact Info' ) }
 					</VerticalNavItem>
 
 					{ ! hasPrivacyProtection && privacyAvailable && (
 						<VerticalNavItem
-							path={ paths.domainManagementPrivacyProtection( this.props.selectedSite.slug, this.props.selectedDomainName ) }>
-							{ this.translate( 'Privacy Protection' ) }
+							path={ paths.domainManagementPrivacyProtection(
+								this.props.selectedSite.slug,
+								this.props.selectedDomainName
+							) }
+						>
+							{ translate( 'Privacy Protection' ) }
 						</VerticalNavItem>
 					) }
 				</VerticalNav>
 			</Main>
 		);
-	},
+	}
 
 	isDataLoading() {
```

Comment:
In other words, if the method is passed as a callback to any component (like `goToEdit` is for `Header`'s `onClick`), then we have to bind that method to `this` (either explicitly through `bind` or using the arrow function notation - I prefer the second option). If it's used just as a regular method (for example, `isDataLoading`) within the same component, no need to bind it to `this`, since `this` will be the current component.